pulp_container/app/viewsets.py

```diff
@@ -836,6 +836,15 @@ class ContainerPushRepositoryViewSet(TagOperationsMixin, ReadOnlyRepositoryViewS
                     "has_namespace_or_obj_perms:container.view_containerpushrepository",
                 ],
             },
+            {
+                "action": ["update", "partial_update"],
+                "principal": "authenticated",
+                "effect": "allow",
+                "condition": [
+                    "has_namespace_or_obj_perms:container.change_containerpushrepository",
```

Comment:
i am struggling to understand whether we need to write a post_migrate signal which will add new permission to the existing objects? on one hand it seems like a good practice so admins who run pulp won't need to manually fix that, on the other hand, this is a new  piece and maybe we should leave this to a deliberate choice of the admins to decide  whether they want to add this new permission?

Reply:
I thought, this is why we introduced the customized flag.